tuf-spec.md

```diff
@@ -1119,21 +1125,29 @@ repo](https://github.com/theupdateframework/specification/issues).
   rollback attack.  On the next update cycle, begin at step 0 and version N of
   the root metadata file.
 
-  * **1.5**. Note that the expiration of the new (intermediate) root metadata
-  file does not matter yet, because we will check for it in step 1.8.
+  * **2.5**. Note that the expiration of the new (intermediate) root metadata
+  file does not matter yet, because we will check for it in step 2.8.
 
-  * **1.6**. **Set the trusted root metadata file** to the new root metadata
+  * **2.6**. **Set the trusted root metadata file** to the new root metadata
   file.
 
-  * **1.7**. **Repeat steps 1.1 to 1.7**.
+  * **2.7**. **Repeat steps 1.1 to 1.7**.
+
+  * **2.8**. **Check for a freeze attack.**
+
+    * **2.8.1** The latest known time should be lower than the update start
+    time plus the update workflow maximum duration.  The update workflow
+    maximum duration is set by the authors of the application using TUF and
+    should typically be the same period at which the repository re-signs
```

Comment:
Also, I wonder if we need to keep repeating the suggestion about setting the update workflow maximum duration.  Might we be better off just saying this once?

Reply:
I wasn't entirely comfortable with this being the timestamp re-signing period. At first it seemed like a reasonable time period that was already defined by the system. However, it's not a period we define or discuss explicitly in the specification. 

Perhaps it makes more sense to have the update workflow maximum duration be explicitly application defined? I'd like it if we suggested a sane default value, but it might be more abstract. Like when we talk about downloading the root metadata and suggest

> up to some W number of bytes (because the size is unknown). The value for W is set by the authors of the application using TUF. For example, W may be tens of kilobytes.

maybe something like

> The latest known time should be lower than the update start time plus the update workflow maximum duration, D.  The value for D is set by the authors of the application using TUF. For example, D may be tens of minutes.

Agreed that it would be better to define the update workflow maximum duration in one place, we can do that at the same time as we record the beginning of the update in step 0. I will make that change.

Reply:
I've removed the explicit suggestion of using the signing period of the timestamp metadata and instead suggested that the time period is application defined.